### PR TITLE
added module and state to handle prodState in zenoss

### PR DESCRIPTION
**Pull request by papoyan**

### CODE REVIEW

File: salt/modules/zenoss.py [hunk 1]
```diff
@@ -159,8 +167,32 @@ def add_device(device=None, device_class=None, collector='localhost'):
...
-    data = dict(deviceName=device, deviceClass=device_class, model=True, collector=collector)
+    data = dict(deviceName=device, deviceClass=device_class, model=True, collector=collector, productionState=prod_state)
     response = _router_request('DeviceRouter', 'addDevice', data=[data])
     return response
+
+
+def set_prod_state(prod_state, device=None):
+    '''
+    A function to set the prod_state in zenoss.
+
```
Comment (jfindlay) [inline salt/modules/zenoss.py]: @papoyan, will you add a `.. versionadded:: Boron` here?  Thanks.

**Comment by jfindlay** [comment]
@papoyan, there is also a [lint error](https://jenkins.saltstack.com/job/salt-pr-lint-n/9187/violations/file/salt/states/zenoss.py/), but replace `,` with `:`.

File: salt/modules/zenoss.py (hunk 1 above)
Comment (papoyan) [inline salt/modules/zenoss.py]: @jfindlay added the version and the whitespace, but two tests are still failing

**Comment by cachedout** [comment]
Thanks @papoyan !

**Pull request merged**